internal/marketplaces/subscriptions/service.go

```diff
@@ -136,7 +136,7 @@ func (s *subscriptionService) CreateProfile(
 		return fmt.Errorf("error while retrieving profile from bundle: %w", err)
 	}
 
-	_, err = s.profiles.CreateProfile(ctx, project.ID, project.Provider, subscription.ID, profile, qtx)
```

Comment:
Check the `ProjectContext` struct in this package (the type of `project` in this method). Now that providers are removed from both rules and profiles, I think we may be able to get rid of that struct, and just pass around a project ID to the methods in this interface.